src/common/components/ReviewModal/ReviewModal.tsx

```diff
@@ -71,11 +76,12 @@ export const ReviewModal = ({
           {/* seeMore link only shown when user is from default reviews page, hidden when in professor/course pages */}
           {seeMore && (
             <div className={seeMoreWrapper()}>
-              <div className={seeMoreLink()}>
-                {/* TODO: link to respective prof/course page */}
-                {/* can use review.professorName for professors and review.courseCode for courses */}
+              <Link
+                href={`${reviewPath.replace(/\s+/g, "-")}`}
+                className={seeMoreLink()}
+              >
                 See more reviews
-              </div>
+              </Link>
```

Comment:
suggestion: the [`<Button/>`](https://github.com/AfterClass-io/afterclass.io-v2/blob/main/src/common/components/Button/Button.tsx) component can be used as a pre-styled `<Link/>` wrapper

eg:

```suggestion
              <Button
                as="a"
                variant="link"
                href=`/professor/${slug}`
                className={seeMoreLink()}
              >
                See more reviews
              </Button>
```